docs/user-guide/getting-started.md

```diff
@@ -68,6 +68,34 @@ sudo service strongbox start
 
 ```
 
+
+```linuxdeb linenums="1" tab="Linux (deb)"
+# Open a terminal
+
+# First, make sure you have a JRE with version 1.8 or greater
```

Comment:
We do not support JRE higher than 1.8 at the moment :)

Reply:
Done, also adjusted the RPM as I based the deb off of it.